.gitignore

```diff
@@ -4,3 +4,4 @@ dist-newstyle
 .DS_Store
 target
 Cargo.lock
+docs
```

Comment:
Why this? `docs` is committed in the repo.

Reply:
I thought that docs should be built locally and not committed